### ISSUE TITLE
React Native: Sample apps

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3349]

**Comment by kuacci** [comment]
@compulim Is there any plan for this sample app in near future?

**Comment by corinagum** [comment]
@kuacci yes, definitely! React Native is not yet supported on Web Chat, though, so samples won't be available until after that update. Currently, there's no definitive timeline for React Native support.

**Comment by kuacci** [comment]
@corinagum Thank you. Is there any other package from BotFramework that can integrate with react native mobile app? Like [botframework-directlinejs](https://www.npmjs.com/package/botframework-directlinejs) ?

**Comment by kuacci** [comment]
I've talked with compulim and got the answer. Thanks for your kindly help.

**Comment by harish-madugula** [comment]
@kuacci hey, did you find any workaround/answer to achieve the above react-native bot framework webchat?
thanks in advance.

**Comment by kuacci** [comment]
@harish-madugula , you may try [this](https://blog.expo.io/so-you-want-to-build-a-chat-app-with-react-native-expo-and-microsofts-bot-framework-4d2327f76ce3) and it works for me.
